holoviews/plotting/util.py

```diff
@@ -1129,6 +1129,8 @@ def hex2rgb(hex):
 
 class apply_nodata(Operation):
 
+    link_inputs = param.Boolean(default=True)
```

Comment:
It could be nice with a doc here.

Reply:
It's inherited from the `Operation` class.